openslides_backend/action/personal_note/create.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict
+
+from ...models.models import PersonalNote
+from ...shared.exceptions import ActionException
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
+from ..register import register_action
+
+
+@register_action("personal_note.create")
+class PersonalNoteCreateAction(CreateAction):
+    """
+    Action to create a personal note.
+    """
+
+    model = PersonalNote()
+    schema = DefaultSchema(PersonalNote()).get_create_schema(
+        required_properties=["content_object_id"],
+        optional_properties=["star", "note"],
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        * Check user_id if anonymous
+        * set user_id from action.
+        * check star or note.
+        * set meeting_id from content_object_id
+        """
+        if self.auth.is_anonymous(self.user_id):
+            raise ActionException("Can't create personal note for anonymous")
+        instance["user_id"] = self.user_id
+        if not (instance.get("star") or instance.get("note")):
+            raise ActionException("Can't create personal note without star or note.")
+        fqid = instance["content_object_id"]
+        content_object = self.datastore.get(fqid, ["meeting_id"])
+        instance["meeting_id"] = content_object.get("meeting_id")
+        return instance
```

Comment:
Check missing that `user_id` and `content_object_id` are unique together -> please add it with test

Reply:
Add a check for (user_id, content_object_id) unique. Add a test.